.cirrus.yml

```diff
@@ -1,21 +1,26 @@
 freebsd_instance:
   image: freebsd-12-0-release-amd64
-freebsd_test_task:
+task:
+  name: FreeBSD
   env:
     matrix:
       - JULIA_SPECIALFUNCTIONS_BUILD_SOURCE: "false"
-        JULIA_VERSION: "1.0"
+        JULIA_VERSION: 1.0
       - JULIA_SPECIALFUNCTIONS_BUILD_SOURCE: "true"
-        JULIA_VERSION: "1.0"
+        JULIA_VERSION: 1.0
+      - JULIA_SPECIALFUNCTIONS_BUILD_SOURCE: "false"
+        JULIA_VERSION: 1.1
+      - JULIA_SPECIALFUNCTIONS_BUILD_SOURCE: "true"
+        JULIA_VERSION: 1.1
+      - JULIA_SPECIALFUNCTIONS_BUILD_SOURCE: "false"
+        JULIA_VERSION: nightly
+      - JULIA_SPECIALFUNCTIONS_BUILD_SOURCE: "true"
+        JULIA_VERSION: nightly
   install_script:
-    - pkg install -y curl
-    - if [ "${JULIA_SPECIALFUNCTIONS_BUILD_SOURCE}" = "true" ]; then pkg install -y gmake gcc; fi
-    - mkdir -p ~/julia
-    - curl -s -L --retry 7 "https://julialang-s3.julialang.org/bin/freebsd/x64/${JULIA_VERSION}/julia-${JULIA_VERSION}-latest-freebsd-x86_64.tar.gz" | tar -C ~/julia -x -z --strip-components=1 -f -
-    - ln -s "${HOME}/julia/bin/julia" /usr/local/bin/julia
-    - julia --color=yes -e "using InteractiveUtils; versioninfo()"
+    - sh -c "$(fetch https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
   build_script:
-    - julia --color=yes -e "using Pkg; Pkg.add(PackageSpec(name=\"SpecialFunctions\", path=pwd()))"
-    - julia --color=yes -e "using Pkg; Pkg.build(\"SpecialFunctions\")"
+    - cirrusjl build
   test_script:
-    - julia --color=yes -e "using Pkg; Pkg.test(\"SpecialFunctions\")"
+    - cirrusjl test
+  coverage_script:
```

Comment:
Can we remove the coverage part? It doesn't actually do anything right now anyway; it always reports success because Coverage.jl doesn't yet support Cirrus. But it looks like it's what's making one of the builds fail.

Reply:
Gone :+1: